### PR TITLE
build: Fix `deb`, `rpm` & `snap` tasks failing

### LINKED ISSUE #2065: `deb`, `rpm` & `snap` tasks failing
I think this is probably a small issue, around quoting things: https://github.com/PRQL/prql/actions/runs/4371758464/jobs/7647949921

I had thought we'd tested them, possibly something small changed after we merged

Would be great to fix them, and then to use them to start publishing release assets — then they would be quite useful. If we can't fix them, we should temporarily disable them until then work.

CC @vanillajonathan (I know I copied you on another issue today — hope that's OK — we're very grateful for your contributions and hope they continue!)

### PR DESCRIPTION
**Pull request by vanillajonathan**
Closes #2065

### CODE REVIEW

**Comment by vanillajonathan** [comment]
@max-sixty `pre-commit.ci` is being mean.

**Comment by max-sixty** [comment]
> @max-sixty `pre-commit.ci` is being mean.

Yeah, it can't fix workflow files, unfortunately ("GitHub prevented pre-commit.ci from autofixing this pr due to autofixes to a workflow file")

Do you have pre-commit locally? Otherwise I can fix...

**Comment by max-sixty** [comment]
One quick way of testing this is to add a `pull_request` trigger for this workflow. We must remember to delete it before we merge though!

As long as it's on a fork (like this PR), it can't do any actual releasing, so it's "safe"

**Comment by vanillajonathan** [comment]
No, I don't have it locally. Please feel free to fix it.

**Comment by max-sixty** [comment]
OK, do you want to try adding the `pull_request` trigger so we can test this? 

Then I can run pre-commit before merging (but also try and get it locally, it should be very tractable to install and will let you be more independent!)

**Comment by max-sixty** [comment]
`deb` & `rpm` look good, as far as they get! 

`snap` still having issues finding the correct path...

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -71,7 +76,7 @@ jobs:
           package_root: .rpmpkg
           maintainer: The PRQL Project
           version: ${{ github.ref }}
```
Comment (max-sixty) [inline .github/workflows/release.yaml]: Try this so we can test on non-tags. I'm not confident it'll work.

```suggestion
          version: ${{ github.ref_type == 'tag' && github.refname || 0 }}
```

**Comment by max-sixty** [comment]
OK great, so that version approach works — worth also copying that to the others. And then we're back to the error around the description https://github.com/PRQL/prql/actions/runs/4410531443/jobs/7728094837#step:6:39

**Comment by max-sixty** [comment]
`rpm` works! Can we get `deb` to pass too?

**Comment by vanillajonathan** [comment]
The multi-line descriptions now works since I reported the issue upstreams and they've since been fixed.

Both the `.deb` and `.rpm` now also builds on PR with the version fallback you provided.

I had some problems with the Snap so now I copy it out from `packages/` to the project root in the GitHub pipeline and run it from the project root and then it works. The job fails fails but this is because secrets are not available in PR so this is not a real problem.

**Comment by max-sixty** [comment]
@vanillajonathan super! Merging, thank you

**Pull request merged**